test/Microsoft.TestPlatform.AcceptanceTests/OrderedTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.AcceptanceTests
+{
+    using System.IO;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class OrderedTests : AcceptanceTestBase
+    {
+
+        /// <summary>
+        /// Ordered Tests created using earlier versions of Visual Studio(i.e. before VS2017) should work fine.
+        /// </summary>
+        [CustomDataTestMethod]
+        [NETFullTargetFramework(inIsolation: true, inProcess: true)]
+        public void OlderOrderedTestsShouldPassWhenAllTestsArePassingTests(RunnerInfo runnerInfo)
```

Comment:
Can we merge these two tests in one?  In this case there will be four test in ordered test (2 pass, 2 fail) and result will be 
pass =2 , fail=1 and Skipped =1.